src/main/java/com/github/LMiljak/DowerTefence/DowerTefence.java

```diff
@@ -0,0 +1,21 @@
+package com.github.LMiljak.DowerTefence;
```

Comment:
Because that's the standard package name convention. com.github.LMiljak is our GroupId, which we can't really change unless we get some sort of other domain (you can do that if you want). DowerTefence is the ArtifactId and the package name convention is {$GroupId}.{$ArtifactId}.<subpackages>.